lib/experimental/class-wp-theme-json-resolver-gutenberg.php

```diff
@@ -115,7 +115,7 @@ public static function get_block_data() {
 
 		// Core here means it's the lower level part of the styles chain.
 		// It can be a core or a third-party block.
-		return new WP_Theme_JSON( $config, 'core' );
+		return new WP_Theme_JSON_Gutenberg( $config, 'core' );
```

Comment:
Why did this change? Cos we've got our Gutenberg version which overloads the Core one?

Reply:
Yeah exactly. Without this the element styles are stripped out because its falling back to the core definitions.